Bottom: acba26b29319bb0c0adbcdc074b564c2a8fbc771
Top:    cb80f2110d69f2d7694e5947dd4ab35fbb0c42d3
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-06-11 00:21:32 +0100

Attempt to debug islandness after inserting hole bug


---

diff --git a/src/pour.c b/src/pour.c
index d4590d0..2100377 100644
--- a/src/pour.c
+++ b/src/pour.c
@@ -1371,8 +1371,12 @@ InitPourClip (DataTypePtr Data, LayerTypePtr layer, PourType * pour)
     }
   while ((pg = tmp) != start_pg);
 
+  printf ("Checking islandiness\n");
   POURPOLYGON_LOOP (pour);
   {
+    printf ("Checking polygon from pour, %s\n",
+            IsPolygonAnIsland (layer, polygon) ? "is an island" :
+                                                 "connected");
     ASSIGN_FLAG (HOLEFLAG, IsPolygonAnIsland (layer, polygon), polygon);
   }
   END_LOOP;
